README.md

```diff
@@ -36,6 +36,14 @@ should be an admin user with passwordless super-user rights.
 
 ## Recipes
 
+### Disable Software Updates
```

Comment:
This was removed in #188

Reply:
Changed by rebasing release/3.0 into this branch and reverting commit.